general-superstaq/general_superstaq/service.py

```diff
@@ -159,6 +159,43 @@ def get_targets(
         )
         return self._client.get_targets(**filters)
 
+    @overload
+    def get_user_info(self) -> dict[str, object]: ...
+
+    @overload
+    def get_user_info(self, *, name: str) -> dict[str, dict[str, object]]: ...
```

Comment:
would it make sense to convert this to `list[dict[str, object]]`, given that the keys of the outer dict are already included within the corresponding values?

(also can `object` be replaced with something like `str | float`?)